src/components/DandisetsPage.vue

```diff
@@ -115,6 +141,8 @@ export default defineComponent({
     // https://next.router.vuejs.org/api/#useroute
     const route = ctx.root.$route;
 
+    const showDrafts = ref(false);
```

Comment:
```suggestion
    const showDrafts = ref(true);
```

Reply:
@satra This will show all draft dandisets with no published versions, including any test dandisets that had data uploaded to them. Is that what you want?

Reply:
@satra This will show all draft dandisets with no published versions, including any test dandisets that had data uploaded to them. Is that what you want?

Reply:
by default, for now yes.

Reply:
once our published count hits 100, we can change the settings. we do need some way of indicating to users that they should use the settings to change display characteristics. perhaps we can do this when we get to a bit more faceted search.